src/promptflow/promptflow/_constants.py

```diff
@@ -164,3 +164,11 @@ class MessageFormatType:
 
 
 DEFAULT_OUTPUT_NAME = "output"
+
+OUTPUT_FILE_NAME = "output.jsonl"
```

Comment:
shall these be kept in promptlfow/_sdk/_constants.py? 

As this storage logic belongs to devkit?

Reply:
Move all these constants to `_devkit_constants.py`.

Reply:
This constants are used by executor server,  which belongs to core.